tools/eslint-excludelist.json

```diff
@@ -47,7 +47,6 @@
 	"projects/plugins/jetpack/_inc/client/state/search/reducer.js",
 	"projects/plugins/jetpack/_inc/client/state/settings/actions.js",
 	"projects/plugins/jetpack/_inc/client/state/settings/reducer.js",
-	"projects/plugins/jetpack/_inc/client/state/site-products/reducer.js",
```

Comment:
Just wondering why this was excluded before, and what the reason is for not excluding it anymore?

Reply:
Those reducers (and other files in that list) follow an old formatting. The idea of having the exclude list is to not having to fix all those files at once. I removed now because I had to touch it and the linter fixed it. The pre-commit doesn't allow us to push if the file has linting errors, and the CI breaks if the file has no error but it's in that list.

Reply:
Good to know! Thanks for explaining